internal/command/machine/run.go

```diff
@@ -342,6 +342,7 @@ func determineImage(ctx context.Context, appName string, imageOrPath string) (im
 			ImageLabel: flag.GetString(ctx, "image-label"),
 			Target:     flag.GetString(ctx, "build-target"),
 			NoCache:    flag.GetBool(ctx, "no-build-cache"),
+			DockerHost: flag.GetString(ctx, "docker-host"),
```

Comment:
same change here

Reply:
@tvdfly what if I put the `docker-host` option under `imgsrc.ImageOptions.Buildpacks` ?
 since it is buildpacks related ?

Reply:
changes has been made, PTAL